build.gradle.kts

```diff
@@ -238,15 +235,15 @@ tasks.withType<RunIdeTask> { maxHeapSize = "4G" }
 
 tasks.withType<RunPluginVerifierTask> {
   val maybeEap =
-      if (intellijVersions["eapOfLatestSupportedMajor"] != null)
+      if (IntellijVersions.eapOfLatestSupportedMajor != null)
           listOf(
-              (intellijVersions["eapOfLatestSupportedMajor"] as String).replace(
+              (IntellijVersions.eapOfLatestSupportedMajor!!).replace(
```

Comment:
Is `!!` needed here? I suppose Kotlin can infer that at this point the value cannot be null

Reply:
Apparently the compiler can deal only with simple statements :(

Reply:
Checker Framework (which is use for null safety in Java) is then much smarter 🧠